Capri.Services/Proposals/ISubmittedProposalGetter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Capri.Services.Proposals
+{
+    public interface ISubmittedProposalGetter
+    {
+        Task<IServiceResult<int>> GetProposalNumber(Guid id);
+        Task<IServiceResult<int>> GetBachelorProposalNumber(Guid promoterId);
+        Task<IServiceResult<int>> GetMasterProposalNumber(Guid promoterId);
```

Comment:
That methods should return number of (in order) proposals and master / bachelor per promoter right? If yes, then i think better naming will be with using plural noun than singular and changing number to `Count`. Like: `GetBachelorProposalsCount`.

Other thing is that in `GetProposalNumber(Guid id);` you didn't specified what id is that and in that method name it can be ambigous.